Equipe 2- Atividade 1

DLRibeiro-zz opened the pull request:
Aqui estão os cenários da atividade 1.

joaoadherval committed:
Criação da Feature de Consultar Aluno

TMotaBastos committed:
Feature de Editar Conceito

rcalegario committed:
Adicionei feature AddStudents

Commit:
Adição da feature "Adicionar critério"

joaoadherval committed:
Término da feature StudentConsult

TMotaBastos committed:
Corrigindo a feature de Editar Conceito

joaoadherval committed:
Merge pull request #1 from joaoadherval/StudentConsult

Student consult

rcalegario committed:
Coloquei feature AddStudents em ingles

TMotaBastos committed:
Colocando a feature em ingles de Editar Conceito

TMotaBastos committed:
Merge pull request #1 from rcalegario/master

Feature AddStudents

Commit:
Tradução da feature "Adicionar critério" para inglês

TMotaBastos committed:
Ajeitando algumas palavras em inglês

DLRibeiro-zz committed:
Feature to add marks to students criteria

Commit:
Merge pull request #1 from DLRibeiro/AddMark

Feature to add marks to students criteria

Commit:
Merge pull request #1 from joaoadherval/StudentConsult

StudentConsult

Commit:
Adição da feature Notificação

Commit:
Merge branch 'master' of https://github.com/Milechwan/TA

Commit:
adicionado arquivo removido no primeiro commit

joaoadherval committed:
Merge pull request #2 from ArthurLapprand/master

2+(4)>>1

joaoadherval committed:
Merge pull request #3 from Milechwan/master

Feature de notificação

DLRibeiro-zz committed:
Merge pull request #2 from joaoadherval/master

Adição de Features

DLRibeiro-zz committed:
Merge pull request #1 from TMotaBastos/master

Cenários de Editar Conceito e Adicionar Estudante

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+  I want to register students in the system
+  So I can evaluate the students with respect to various criteria
+
+#Cenário Controler
```
"Controller" seria o correto

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+    Then all the marks will not be stored in on the "X" criteria's history of each student
+  Scenario: Add mark more than once with same origin
+    And marks for every student on the "X" criteria, originated form "Test" and dated from "28/03/2016" are already in the system
+    When I want to add a mark to all students to a the "X" criteria, without a specific origin and dated from "28/03/2016"
```
O correto seria tirar o "a" antes de "the criteria"

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+    When I want to add a mark to all students to a the "X" criteria, without a specific origin and dated from "28/03/2016".
+    Then all the marks will not be stored in on the "X" criteria's history of each student
+  Scenario: Add mark more than once with same origin
+    And marks for every student on the "X" criteria, originated form "Test" and dated from "28/03/2016" are already in the system
```
Trocar "form" por "from"

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+    Then all the marks will not be stored in on the "X" criteria's history of each student
+
+  Scenario: Error related to add a repetead mark
+    And I am at the "Add concept" screen
```
Dois "and" antes do "then" mas nenhum "given". Substituir esse "and" pelo "given" seria o correto (devido ao próprio padrão)

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+  Scenario: Error related to add a repetead mark
+    And I am at the "Add concept" screen
+    And there already are marks for the "X" criteria, originated from "Test" and dated from "24/03/2016" in the system
+    When I want to add a mark to all students to a the "X" criteria, without a specific origin and dated from "28/03/2016".
```
"want to" parece não ser uma ação de fato. Trocando por "try to" deixaria explícita a ação

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+    Then an error message related to trying to add a repetead mark will be displayed
+
+  Scenario: Import marks
+    And I organized all marks for the "X" criteron originated from "Midterm", dated from "31/03/2016" in a spreedsheet
```
Deve-se trocar "criteron" por "criterion" ou "criteria"

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+  Scenario: Register a new student
+    Given the student "Roberto Alves" with login "ra" is not registered in the system
+    When I register "Roberto Alves" with login "ra"
+    Then the student "Roberto Alves" with login "rc" is registered in the system
```
"rc" deveria ser "ra"

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+
+
+  Scenario: Import repeated marks
+    And I organized all marks for the "X" criteria originated from "Midterm", dated from "31/03/2016" in a spreedsheet
```
Dois "and" antes do "then" mas nenhum "given". Substituir esse "and" pelo "given" seria o correto (devido ao próprio padrão)

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+  Scenario: Register a student twice
+    Given the student "Roberto Alves" with login "ra" is registered in the system
+    When I register "Roberto Alves" with login "ra"
+    Then the system do nothing
```
the system DOES nothing

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+#GUI Scenario
+  Scenario: Search the criteria's average with just a part of the student's name
+    Given I'm on the "Alunos" page
+    And the students "Jose Maria"  e "Jose da Silva" are registered in the system
```
Deveria ser "Jose Maria" and "Jose da Silva".
Além disso, mistura um Cenário de GUI com um Cenário de Controlador.
Deveria ser algo do tipo "And I can see the student "Jose Maria" and the student "Jose da Silva" in the list of students".

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+    Given I'm on the "Alunos" page
+    And the students "Jose Maria"  e "Jose da Silva" are registered in the system
+    When I search for "Jose"
+    Then a list with the names of "Jose Maria" e Jose sa Silva" will appear on the screen
```
Deveria ser "Jose Maria" and "Jose da Silva"

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+    As an administrator
+    I want to be notified about the students issues without searching for their historic
+    So I can know who has its performance under the class average quickly
+
```
Correctly follows the Feature template which introduces the User Story. Just put "class average grades", because the focus is on the students average grades

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+Feature: Register Student
```
Checklist:
- [ OK ] Scenarios should reflect the stakeholders needs
- [ NOT OK ] Scenario description should strictly correspond to scenario semantics
- [ OK ] Look for short scenario steps sentences (use And as needed)
- [ OK ] Do not duplicate scenario steps
- [ OK ] Do not duplicate scenarios
- [ OK ] Scenario should not be vague or incomplete
- [ OK ] Specify actions, not non actions or empty actions
- [ NOT OK ] Avoid language errors
- [ OK ] Use high level actions
- [ OK ] Write platform independent scenarios
- [ OK ] Focus on testing situation, direct to the point
- [ OK ] Focus on testing situation, not general requirement
- [ NOT OK ] Parametrize scenarios

rcalegario commented on test/cucumber/EditarConceito.feature:
```diff
@@ -0,0 +1,28 @@
+#GUI Scenario
+Scenario: Edit an evaluation for a non existent one
+  Given there is a student with the following informations: student "Marcos Antônio", login "ma2", has a "MANA" evaluation in the criterion "X", from "Prova 2", date "11/11/11"
+  When I request the system to modify the evaluation "MANA" to "10"
```
é necessário dizer todas as informação sobre a avaliação para não haver problemas na hora do teste
Correção: I request the system to modify the evaluation "MANA" to "10" in the criterion "X", from "Prova 2", date "11/11/11"

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+
+#Controller
+  Scenario: Add new report type to the reports list
+    Given the "70% of grades are MANA" and �70% of grades are under the class average� are on the system
```
"that" instead of "the", also quotation marks formation problem

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+#Controller
+  Scenario: Add new report type to the reports list
+    Given the "70% of grades are MANA" and �70% of grades are under the class average� are on the system
+    When I add the concept "MANA" to the student "Silvio", login "shd2"
```
As a Controller Scenario, mentions to abstractions or navigation should be avoided, focusing on the systems perspective

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+    When I add the concept "MANA" to the student "Silvio", login "shd2"
+    And his grades become 70% of MANA
+    Then the report "70% of grades are MANA" is added to the reports list
+
```
The report should be updated since they already are in the system

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+    When I add the concept "MANA" to the student "Joan", login "jsd"
+    And his grades become 70% of MANA
+    Then I should see a new notification related to the new report type
+
```
Just language errors

rcalegario commented on test/cucumber/EditarConceito.feature:
```diff
@@ -0,0 +1,28 @@
+Scenario: Edit an evaluation for a non existent one
+  Given there is a student with the following informations: student "Marcos Antônio", login "ma2", has a "MANA" evaluation in the criterion "X", from "Prova 2", date "11/11/11"
+  When I request the system to modify the evaluation "MANA" to "10"
+  Then an error message related to the evaluation pops up
```
Pops up é uma instrução de baixo nível, podendo ser substituida por appear

joaoadherval commented on test/cucumber/Notificacao.feature:
```diff
@@ -0,0 +1,32 @@
+Scenario: Show the students list in the "70% of grades are under the class average" report
+  Given I am in the Report page
+  When I select the "70% of grades are under the class average report"
+  Then I should see the students list related to the "70% of grades are under the class average" report
```
In general, the scenarios reflect the stackholders needs, short ones, using "And" when needed, without duplicate or vague or incomplete scenarios. Specified actions, platform independent scenarios

DLRibeiro-zz commented on test/cucumber/AdicionarCritério.feature:
```diff
@@ -0,0 +1,32 @@
+#Arthur Lapprand
```
Checklist:
Scenarios should reflect the stakeholders needs - OK
Scenario description should strictly correspond to scenario semantics - OK
Look for short scenario steps sentences (use And as needed) - OK
Do not duplicate scenario steps - OK
Do not duplicate scenarios - OK
Scenario should not be vague or incomplete - OK
Specify actions, not non actions or empty actions - OK
Avoid language errors - OK
Use high level actions - OK
Write platform independent scenarios - OK
Focus on testing situation, direct to the point - OK
Focus on testing situation, not general requirement - OK

rcalegario commented on test/cucumber/EditarConceito.feature:
```diff
@@ -0,0 +1,28 @@
+#Controller Scenario
+Scenario: Edit evaluation of a student
+  Given there is a student with the following information: student "Marcos Antônio", login "ma2", has a "MANA" evaluation in the criterion "X", from "Prova 2", date "11/11/11"
+  When I modify the "MANA" evaluation to "MA" in the criterion "X", from "Prova 2", date "11/11/11"
```
É importante deixar sempre claro de quem é a nota que vai ser modificada. Nesse caso deveria colocar as informações do estudante

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+#João Adherval
```
Checklist:
Scenarios should reflect the stakeholders needs - OK
Scenario description should strictly correspond to scenario semantics - NOT OK
Look for short scenario steps sentences (use And as needed) - OK
Do not duplicate scenario steps - OK
Do not duplicate scenarios - OK
Scenario should not be vague or incomplete - OK
Specify actions, not non actions or empty actions - OK
Avoid language errors - NOT OK
Use high level actions - OK
Write platform independent scenarios - OK
Focus on testing situation, direct to the point - OK
Focus on testing situation, not general requirement - OK

Cons de todos as features:
- Todo mundo usou nomes diferentes para "nota"
- João e Milena - grades
- Thiago - evaluation
- Danilo - marks

DLRibeiro-zz commented on test/cucumber/AdicionarCritério.feature:
```diff
@@ -0,0 +1,32 @@
+    Given the criterion named "P1" already exists on the system
+    When I create the criterion "P1"
+    Then system does nothing
+
```
Parametrizar o nome das páginas.

milenascc commented on test/cucumber/AddMark.feature:
```diff
@@ -0,0 +1,33 @@
+
+Feature AddMark
+  As a teacher, I want to give marks referring to each criteria to each student, to show them their progress in class
```
Trocaria o "to show(...)" por "So I can show(...)" para respeitar o padrão de descrição de feature

DLRibeiro-zz commented on test/cucumber/AdicionarCritério.feature:
```diff
@@ -0,0 +1,32 @@
+    When I add the criterion "P1"
+    And the criterion "P1" already exists
+    Then I should see a message related to the criterion registration failure
+
```
And(line 24), deveria ser na parte do given, pois é um estado do sistema

DLRibeiro-zz commented on test/cucumber/AdicionarCritério.feature:
```diff
@@ -0,0 +1,32 @@
+    And the criterion "P1" already exists
+    Then I should see a message related to the criterion registration failure
+
+#GUI Scenario
```
Nome poderia ser parametrizado, pois podem existir vários campos para preenchimento.

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+    Given I'm on the "Alunos" page
+    And the student "Jose Maria" with login "jm" is registered
+    When I request the student's information
+    Then all the student's average grades in all concepts will appear in the screen
```
Em vez de "concepts" deveria ser "criteria", visto que a ideia de conceito foi ligada a "grades"

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+    Then all the student's average grades in all concepts will appear in the screen
+
+#GUI Scenario
+  Scenario: Consult a non registered student's average grade concepts
```
Mesmo problema de troca entre "concepts" e "criteria"

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+    Given I'm on the "Alunos" page
+    And the student "Jose Maria" with login "jm" is not registered
+    When I request the student's information
+    Then a error message will appear in the screen
```
Deveria ser "an" no lugar de "a"

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+#GUI Scenario
+  Scenario: Consult a student's average grade concepts
+    Given I'm on the "Alunos" page
+    And the student "Jose Maria" with login "jm" is registered
```
Mistura um Cenário de GUI com um Cenário de Controlador.
Deveria ser algo do tipo "And I can see the student "Jose Maria" with login "jm" in the list of students"

TMotaBastos commented on test/cucumber/StudentConsult.feature:
```diff
@@ -0,0 +1,39 @@
+#GUI Scenario
+  Scenario: Consult a non registered student's average grade concepts
+    Given I'm on the "Alunos" page
+    And the student "Jose Maria" with login "jm" is not registered
```
Mistura um Cenário de GUI com um Cenário de Controlador.
Deveria ser algo do tipo "And I can't see the student "Jose Maria" with login "jm" in the list of students"

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+#Cenário GUI
+  Scenario: Error message when registering a student twice
+    Given  I am in the add student page
+    And the student "Roberto Alves" with login "ra" is registered in the system
```
Mistura Controller com GUI, sugestão:
And eu vejo o aluno "Roberto Alves" na lista de alunos cadastrados

lapprand commented on test/cucumber/AddStudents.feature:
```diff
@@ -0,0 +1,32 @@
+
+#Cenário GUI
+  Scenario: Error message when registering a student twice
+    Given  I am in the add student page
```
"add student" page - também parametrizar no outro Cenário de GUI

Commit:
Correções:
- parametrização do nome da página em GUI Scenarios
- troca de localização do AND

Commit:
Notificação atualizada com base na revisão

joaoadherval committed:
StudentConsult.feature agora está correta de acordo com o que foi corrigdo pelo aluno tmb2

DLRibeiro-zz committed:
Update and rename AddMark.feature to AddEvaluation.feature

DLRibeiro-zz committed:
Mudei pra evaluation

DLRibeiro-zz committed:
Mudei pra evaluation

DLRibeiro-zz committed:
Mudei pra evaluation
Add Evaluation Feature

Commit:
Tradução do nome da feature.

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

DLRibeiro-zz committed:
Muita coisa

rcalegario committed:
Esquele do Steps de AddStudentes criado

Commit:
isn't -> is not

TMotaBastos committed:
Corrigindo os problemas vistos na correção

rcalegario committed:
Steps de AddStudentes implementados

Commit:
Scenarios de controllers feitos

Commit:
Mudando de volta removendo as aspas.

Commit:
Pequenas mudanças

Commit:
Adicionado scenario de GUI

TMotaBastos committed:
Codigo de implementação de um teste de controlador e um teste de GUI

Commit:
Notificação atualizada para correção de erros de linguagem e criação do teste

rcalegario committed:
Merge pull request #1 from ArthurLapprand/master

Pull request dos Testes

DLRibeiro-zz committed:
Merge pull request #3 from Milechwan/master

Pull request para segunda entrega - testes

DLRibeiro-zz committed:
Merge pull request #4 from DLRibeiro/TesteFeature

Teste feature

Commit:
Merge pull request #2 from rcalegario/master

Pull Request Testes

joaoadherval committed:
Testes da Feature StudentConsult criada

joaoadherval committed:
Merge pull request #4 from TMotaBastos/master

Pull Request da segunda entrega

TMotaBastos committed:
Merge pull request #3 from joaoadherval/master

Criação dos Testes

DLRibeiro-zz committed:
Merge pull request #5 from pauloborba/master

Recreate some files

TMotaBastos committed:
Merge branch 'master' of https://github.com/DLRibeiro/TA into DLRibeiro-master

merge do pull request

DLRibeiro-zz committed:
Ajustei minha feature

Commit:
Commit de todos os arquivos alterados para resolução de conflito

DLRibeiro-zz committed:
Ajustei minha feature

Commit:
Commit de todos os arquivos alterados para resolução de conflito (quarta vez)

Commit:
Adicionando controllers

Commit:
Adicionado CreateCriterionPage e outras pequenas mudanças

Commit:
Modificações na classe de teste e na feature para tentar rodar os testes

DLRibeiro-zz committed:
Merge pull request #6 from Milechwan/master

Só atualizando tudo pro meu projeto

Commit:
Mudanças feitas na monitoria 12/05

Commit:
pequenas mudanças

Commit:
Troca de findBy no Steps por método de CriterionTestDataAndOperations que chama método do CriterionController

DLRibeiro-zz committed:
Merge remote-tracking branch 'origin/master'

DLRibeiro-zz committed:
Ajustei minhas classes dominio

DLRibeiro-zz committed:
Ajustei alguns métodos.

DLRibeiro-zz committed:
Update StudentController.groovy

joaoadherval committed:
Merge pull request #5 from Milechwan/master

Atualização

DLRibeiro-zz committed:
Novas domains e view e controller associado

DLRibeiro-zz committed:
Merge remote-tracking branch 'origin/master'

DLRibeiro-zz committed:
Algumas mudanças por causa de closures

DLRibeiro-zz committed:
FormattedDate no StudentController

DLRibeiro-zz committed:
FormattedDate no StudentController

DLRibeiro-zz committed:
Novas operações pros testes

DLRibeiro-zz committed:
Novas operações pros testes

DLRibeiro-zz committed:
Mudanças em muita coisa

DLRibeiro-zz committed:
Mudanças em muita coisa

Commit:
Recriação do CriterionController

Commit:
Correção de bug do IntelliJ

Commit:
Tentando ajeitar mais bugs do IntelliJ

Commit:
Tentando corrigir bug do IntelliJ parte 2

Commit:
Corrigindo Bugs IntelliJ parte 3

Commit:
Acoplamento de métodos no controlador

DLRibeiro-zz committed:
Novo método de controller, e novos construtores para classes básicas

DLRibeiro-zz committed:
Novo método de controller, e novos construtores para classes básicas

Commit:
2 testes passam, 1 falha

joaoadherval committed:
Testes da Feature StudentConsult criada

joaoadherval committed:
Test's functions implemented, looking to test it. Can't merge with atualizations

DLRibeiro-zz committed:
Alterei minha feature, não consegui terminar tudo

Commit:
Modificações na classe de teste , na feature, controladores e testanddataoperations

Commit:
2 testes passam, 1 falha - tentando configurar o chromedriver

Commit:
Commit para resolver conflitos

Commit:
Merge branch 'master' of https://github.com/DLRibeiro/TA into DLRibeiro-master

# Conflicts:
#	grails-app/domain/ta/Evaluation.groovy
#	grails-app/domain/ta/Student.groovy
#	test/functional/steps/EvaluationdDataAndOperations.groovy

Commit:
Merge branch 'DLRibeiro-master'

# Conflicts:
#	grails-app/controllers/ta/CriterionController.groovy
#	grails-app/controllers/ta/StudentController.groovy
#	grails-app/domain/ta/Criterion.groovy
#	grails-app/domain/ta/EvaluationsByCriterion.groovy

Commit:
Merge branch 'refs/heads/pr/2'

# Conflicts:
#	grails-app/controllers/ta/CriterionController.groovy
#	grails-app/controllers/ta/StudentController.groovy
#	grails-app/domain/ta/Criterion.groovy
#	grails-app/domain/ta/Student.groovy
#	grails-app/views/criterion/_form.gsp
#	grails-app/views/criterion/index.gsp
#	grails-app/views/criterion/show.gsp
#	grails-app/views/student/index.gsp
#	test/cucumber/steps/EvaluateStudentSteps.groovy
#	test/functional/steps/EvaluateStudentTestDataAndOperations.groovy

Commit:
2 testes passam, 1 falha - tentando configurar o chromedriver

Commit:
Merge branch 'master' of https://github.com/ArthurLapprand/TA

DLRibeiro-zz committed:
Setei as variaveis iniciais agora

Commit:
Resolvendo conflitos com master de João

Commit:
merge

Commit:
Merge pull request #3 from DLRibeiro/master

Setei as variaveis iniciais agora

Commit:
Adições de métodos e reparação na classe de testes

Commit:
Merge para pull request

DLRibeiro-zz committed:
Corrigi erros de chamada de método

Passava Date, ao invés da String da data

DLRibeiro-zz committed:
Consertei chamada de métodos.

de addEvaluations para addEvaluationAllStudents()

DLRibeiro-zz committed:
Consertei nome de método com nome incorreto

De addEvaluationAllStudentes para addEvaluationAllStudents

DLRibeiro-zz committed:
Alterei a função

DLRibeiro-zz committed:
Mudei createEvaluation(com parametros) para createEvaluation()

DLRibeiro-zz committed:
Criterion passou a ser string

Para eliminar dependência, criterion agora é string

DLRibeiro-zz committed:
Corrigi código referente a evaluation

Já que criterion em e evaluation não é mais objeto, preciso modificar como ele é construido

DLRibeiro-zz committed:
Mudanças referentes a Evaluation

Mudanças por criterion ser string não classe

DLRibeiro-zz committed:
Update EvaluationdDataAndOperations.groovy

DLRibeiro-zz committed:
Mudanças referentes a Evaluation

DLRibeiro-zz committed:
Update EvaluationdDataAndOperations.groovy

rcalegario committed:
Testes funcionando, só com os meus arquivos (sem nenhum pull request"

DLRibeiro-zz committed:
Corrigindo coisas do test, e agora Student coloca todos os criterios na criação do objeto

rcalegario committed:
Modificacao no gitignore

rcalegario committed:
Adicionando tudo de AddStudent

Commit:
Sem GebConfig

DLRibeiro-zz committed:
O teste roda, mas falha

DLRibeiro-zz committed:
Correção de nome de variavel

rcalegario committed:
merge com milena

joaoadherval committed:
Efetuação dos testes de controller bem sucedidas

rcalegario committed:
view search student with login

Commit:
transferência do projetoToy pra cá

Commit:
adição de unversioned files no gitignore

joaoadherval committed:
Merge branch 'master' of https://github.com/rcalegario/TA into rcalegario-master

Conflicts:
	.idea/modules.xml
	grails-app/controllers/ta/CriterionController.groovy
	grails-app/controllers/ta/StudentController.groovy
	grails-app/domain/ta/Criterion.groovy
	grails-app/domain/ta/EvaluationsByCriterion.groovy
	grails-app/domain/ta/Student.groovy
	grails-app/views/criterion/index.gsp
	grails-app/views/criterion/show.gsp
	grails-app/views/student/index.gsp
	test/cucumber/EvaluateStudent.feature
	test/cucumber/RegisterEvaluation.feature
	test/cucumber/StudentConsult.feature
	test/cucumber/steps/EvaluateStudentSteps.groovy
	test/cucumber/steps/RegisterEvaluation.groovy
	test/cucumber/steps/StudentConsultSteps.groovy
	test/functional/steps/StudentConsultTestDataAndOperations.groovy

Commit:
inspeção de código

Commit:
complementação de testes

DLRibeiro-zz committed:
Corrigi algumas redundancias

Commit:
Criação de views, modificação no controller e na domain class de Report

joaoadherval committed:
Erro nos testes. Botao da View de Search agora funciona. Falta listar alunos

rcalegario committed:
Textinho do commit

Commit:
-added single student average calculation method
-fixed single student average shown when importing group of students

joaoadherval committed:
View search funcionando. Consulta por login apresenta os resultados e trata erro de estudante não existente

joaoadherval committed:
Search funcionando com nomes. Falta fazer com que funcione com os dois, nome e login

joaoadherval committed:
Search funcionando com nomes e logins

joaoadherval committed:
Commit sem pastas bugadas yaay

joaoadherval committed:
Atualizando gitignore

rcalegario committed:
modificacao em um .idea

rcalegario committed:
merge com joao

rcalegario committed:
finalizando merge com joao

Commit:
-bug -> Student not listing correctly

Commit:
-gitignore corrected

Commit:
Merge com Calega

Commit:
Merge branch 'rcalegario-master'

Commit:
-fixed Student criteria and average listing
-fixed searchStudent and importStudents html and css design
-fixed various other controller issues such as refactoring attribute names

Commit:
Commit para dar merge com Lapprand

Commit:
Commit para dar merge com Lapprand

Commit:
Commit para dar merge com Lapprand

Commit:
Funcionamento da feature de Notificação ok (relatórios atualizam ao se adicionar um Evaluation)

DLRibeiro-zz committed:
Corrigi problemas no teste

Commit:
-added link between showStudent and showEvaluationsByCriterion
-fixed bug that caused student Global Average to not update
-fixed bug that caused student evaluationsByCriterion Criterion Average to not update
-all is beautiful (except delete)

Commit:
Merge com Milena

Commit:
Merge branch 'Milechwan-master'

DLRibeiro-zz committed:
Tirei alguns comentários

Commit:
-fixed some GUI issues
-made GUI tables more intuitive

Commit:
-up with Danilo changes

DLRibeiro-zz committed:
Reajustei a feature

DLRibeiro-zz committed:
Adicionei scenario de GUI

DLRibeiro-zz committed:
Mudei algumas funções

Commit:
-fixed Evaluation delete

DLRibeiro-zz committed:
comentario

DLRibeiro-zz committed:
Update AddEvaluation.groovy

DLRibeiro-zz committed:
Resolvendo conflito

DLRibeiro-zz committed:
FINAL TRY

DLRibeiro-zz committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	test/cucumber/steps/AddEvaluation.groovy

DLRibeiro-zz committed:
Corrigindo testes, e agora o saveAll() do EvaluationController verifica se a avaliação foi criado repetidas vezes

DLRibeiro-zz committed:
Esqueci de incluir o dataAndOperations

DLRibeiro-zz committed:
Testes de controller todos funcionando

DLRibeiro-zz committed:
Coloquei algumas pages

DLRibeiro-zz committed:
Mais pages pra teste

DLRibeiro-zz committed:
Ajustes

joaoadherval committed:
Student e Student Controller atualizados. Scenarios de Controllador funcionando

joaoadherval committed:
Scenarios de GUI funcionando

joaoadherval committed:
Refatoração do Controlador de Student, Steps, Page e Feature

DLRibeiro-zz committed:
Update AddEvaluationPage.groovy

DLRibeiro-zz committed:
Update AddEvaluation.groovy

DLRibeiro-zz committed:
Merge remote-tracking branch 'origin/master'

DLRibeiro-zz committed:
REFACTORING DE MÉTODOS SEM UTILIDADE, E DE COMPORTAMENTO REPLICADO

DLRibeiro-zz committed:
REFACTORING DE MÉTODOS INUTEIS

DLRibeiro-zz committed:
Merge branch 'master' of git://github.com/joaoadherval/TA into joaoadherval-master

DLRibeiro-zz committed:
Tirando alguns heads de conflito

DLRibeiro-zz committed:
Refatoração de métodos inuteis

DLRibeiro-zz committed:
Mudei a maneira de selecionar o dia

DLRibeiro-zz committed:
Mudei um teste
